packages-proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -18,7 +18,7 @@
         "yellow": { "value": "#ffe600" }
```

Comment:
The logo texts still use `#000`, should those be changed as well?

Reply:
No, we can’t touch the logo.